docs/book/misc/debugging.md

```diff
@@ -0,0 +1,234 @@
+---
+description: A guide to debug common issues and get help.
+---
+
+# Debugging Guide
+This page documents a series of how-tos for efficient debugging and getting help without losing your hair.
```

Comment:
`without losing your hair` maybe isnt the right tone. How about `efficient debugging and getting help.`

Reply:
Fixed.